libs/designsystem/src/lib/components/page/page.component.spec.ts

```diff
@@ -171,9 +171,9 @@ describe('PageComponent', () => {
       expect(leaveEmitSpy).toHaveBeenCalledTimes(1);
     });
   });
-  function triggerOnLeave(router: SpyObject<Router>) {
+  async function triggerOnLeave(router: SpyObject<Router>) {
```

Comment:
Don't really see the need for this helper method: spectator.detectChanges() should be run by each test (e.g. in a beforeEach) to setup nav listener. Then it's just a matter of calling router.navigate(...). The 2nd `spectator.detectChanges` doesn't do anything...

Reply:
It explains the intent

Reply:
Ok check 👍